### ISSUE TITLE
Make website available offline

### ISSUE THREAD

**Issue by tachyons** [IEEEKeralaSection/rescuekerala#56]
Internet connection can be unreliable in flood affected areas. So making static pages offline may help in such cases

It is easy with libraries like this https://github.com/TalAter/UpUp

**Comment by jose2007kj** [comment]
https://medium.com/beginners-guide-to-mobile-web-development/convert-django-website-to-a-progressive-web-app-3536bc4f2862

**Comment by sreeramvenkitesh** [comment]
wont it be better to have a csv downloaded when clicked on a  button and thus helping them ?

**Comment by tachyons** [comment]
@jose2007kj Above linked library also does the same internally, I think it is better to have a simple implementation to begin with. So that it won't break rest of the code

@sreeramvenkitesh CSV files are relatively difficult to open and read from mobile, having an html page is much better

**Comment by jose2007kj** [comment]
you are right @tachyons ......

**Comment by sreeramvenkitesh** [comment]
@tachyons  fair point, what about having an option to download an image with all the details so that users dont even have to open the browers in these cases  ?

**Comment by naveenpf** [comment]
Complex to take up

**Comment by biswaz** [comment]
Will the library overhead increase bandwidth noticeably?

No promises on implementing

**Comment by tachyons** [comment]
Less than 3kb
